src/main/java/pluto/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package pluto.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDateTime by;
```

Comment:
I feel mixed about the name of this variable. It is technically enough, but I still feel like it should be more descriptive. Could you consider making it more descriptive?

(This also applies to the similar private variable in Event)